skytemple_files/graphics/chara_wan/writer.py

```diff
@@ -0,0 +1,1412 @@
+import sys
+import os
+import glob
+from PIL import Image
+import xml.etree.ElementTree as ET
+import xml.dom.minidom as minidom
+from chara_wan.model import WanFile, SequenceFrame, MetaFramePiece, FrameOffset, AnimStat, ImgPiece, \
+    MINUS_FRAME, DIM_TABLE, TEX_SIZE
+import chara_wan.utils as exUtils
+
+
+
+
+DRAW_CENTER_X = 0
+DRAW_CENTER_Y = -4
+
+MAX_ANIMS = 44
+
+ANIM_ORDER = [0, 5, 6, 7, 8, 9, 10, 11, 12, 1, 2, 3, 4, 13, 14, 15, 16, 17, 18, 19, 20, \
+              21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43]
+
+DEBUG_PRINT = False
+
+
+
+
+
+
+def ExportWan(out_file, wan):
```

Comment:
Please make this return a byte array instead / work on a bytarray. You can use the `io` module if you want to use pseudo files.
Otherwise the `util` module has utility functions for working with byte arrays:
https://github.com/SkyTemple/skytemple-files/blob/master/skytemple_files/common/util.py